packages/commonwealth/client/scripts/state/api/threads/createThread.ts

```diff
@@ -84,6 +93,16 @@ const useCreateThreadMutation = ({
               : totalThreadsInCommunityForVoting,
         }),
       );
+
+      if (userOnboardingEnabled) {
+        const profileId = app?.user?.addresses?.[0]?.profile?.id;
+        setCardTempMarkedAsCompleted(UserTrainingCardTypes.CreateContent);
+        setShouldHideTrainingCardsPermanently(
```

Comment:
you use this pair multiple times, how about creating another function that would wrap those two, so you would need to invoke just one instead of two?